Core/Src/subsystems/gimbal.cpp

```diff
@@ -58,18 +60,10 @@ void act() {
 }
 
 double calculateAngleError(double currAngle, double targetAngle) {
-    // IMPLEMENT THIS
-    /* 
-    Given a current angle value (currAngle, in radians) and a target angle value (targetAngle, in radians):
-        - Find and return the shortest angle error in radians with direction.
```

Comment:
We missed this line initially; we've updated the pull request with a fix (and the fix is also slightly more efficient for the common case).